docs/sources/panels/inspect-panel.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Inspect a panel"
+type = "docs"
+[menu.docs]
+identifier = "inspect-a-panel"
+parent = "panels"
+weight = 400
++++
+
+# Inspect a panel
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
+
+The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated variable (CSV) file, view query requests, and export panel and data JSON.
+
+## Panel inspector UI
+
+The panel inspector displays Inspect: <NameOfPanelBeingInspected> at the top of the pane. Click the arrow in the upper right corner to expand or reduce the pane.
+
+The panel inspector consists of four tabs:
+
+* **Data tab -** Shows the raw data returned by the query with transformations applied. Field options such as overrides and value mappings are not applied.
+* **Stats tab -** Shows how long your query takes and how much it returns.
+* **JSON tab -** Allows you to view and copy the panel JSON, panel data JSON, and data frame structure JSON. This is useful if you are provisioning or administering Grafana. 
+* **Query tab -** Shows you the requests to the server sent when Grafana queries the data source. 
+
+> **Note:** Not all panel types include all four tabs. For example, dashboard list panels do not have raw data to inspect, so they do not display the Stats, Data, or Query tabs.
+
+## Panel inspector tasks
+Tasks you can perform in the panel inspector are described below. 
+
+### Open the panel inspector
+
+You can inspect any panel that you can view.
+
+1. In Grafana, navigate to the dashboard that contains the panel you want to inspect.
+1. Click the title of the panel you want to inspect and then click **Inspect**.
+   Or
+   Hover your cursor over the panel title and then press **i**.
+
+The panel inspector pane opens on the right side of the screen.
+
+### Inspect raw query results
+
+View raw query results in a table. This is the data returned by the query with transformations applied and before the panel applies field configurations or overrides.
+
+1. Open the panel inspector and then click the **Data** tab or in the panel menu click **Inspect > Data**.
+
+2. If your panel contains multiple queries or queries multiple nodes, then you have additional options.
```

Comment:
```suggestion
2. If your panel contains multiple queries, then you have additional options.
```

Reply:
I got the Join option on a panel with a single query in Play. 
https://play.grafana.org/d/000000012/grafana-play-home?editPanel=2&orgId=1

Reply:
So the "requires multiple queries" bit is not true.

Reply:
But "nodes" might not be accurate, either.